app/models/manageiq/providers/kubernetes/container_manager/metrics_capture/hawkular_client_mixin.rb

```diff
@@ -7,11 +7,12 @@ def hawkular_client
 
   def hawkular_entrypoint
     hawkular_endpoint = @ext_management_system.connection_configurations.hawkular.try(:endpoint)
+    hawkular_endpoint_empty = hawkular_endpoint.nil? || hawkular_endpoint.hostname.empty?
```

Comment:
@simon3z it's two different objects the first is `hawkular_endpoint` and then `hawkular_endpoint.hostname`.